lib/dialects/sqlite3/schema/internal/sqlite-ddl-operations.js

```diff
@@ -1,21 +1,34 @@
 const identity = require('lodash/identity');
 const chunk = require('lodash/chunk');
 
-async function insertChunked(trx, chunkSize, target, iterator, result) {
+function insertChunked(trx, chunkSize, target, iterator, existingData) {
+  const result = [];
   iterator = iterator || identity;
-  const chunked = chunk(result, chunkSize);
+  const chunked = chunk(existingData, chunkSize);
   for (const batch of chunked) {
-    await trx.queryBuilder().table(target).insert(batch.map(iterator));
+    result.push(
+      trx.queryBuilder().table(target).insert(batch.map(iterator)).toQuery()
+    );
   }
+  return result;
 }
 
 function createTempTable(createTable, tablename, alteredName) {
   return createTable.sql.replace(tablename, alteredName);
 }
 
 async function copyData(trx, tableName, alteredName) {
```

Comment:
`copyData` and `reinsertData` will be removed in the end, we'll use `copyAllData` for everything, as it's significantly better for large amount of data.